notification-component-icon/src/NotificationIcon.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {
+  Dropdown,
+  DropdownMenu,
+  DropdownToggle,
+  DropdownItem,
+} from 'reactstrap';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+
+class NotificationIcon extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dropdownOpen: false,
+      notifications: [
```

Comment:
For the sake of making the path forward clearer, I'd love it if this sample JSON were in a separate file that the component pulls in with `fetch`.  We can start with static JSON available at a relamtive URL;  when we're ready, we change the URL to an API endpoint provided by the Spring Boot portion of this project.

Reply:
Sure, adding demo json

Reply:
Added :ballot_box_with_check: